Source/WebCore/dom/CustomElementRegistry.idl

```diff
@@ -31,6 +31,7 @@
 ] interface CustomElementRegistry {
     [CEReactions, Custom] undefined define(DOMString name, Function constructor);
     any get([AtomString] DOMString name);
+    [Custom] DOMString? getName(any constructor);
```

Comment:
```suggestion
    [Custom] DOMString? getName(Function constructor);
```

Reply:
Hm... this results in an error like this:
```
Could NOT find IDL file for interface "Function"!
Could NOT find IDL file for interface "Function"!